goal_src/jak2/engine/draw/drawable.gc

```diff
@@ -1144,8 +1142,11 @@
                       )
                     ;; lod hacks!
                     (with-pc
-                      (if (not (-> *pc-settings* ps2-lod-dist?))
-                        (set! lod-idx (minmax (-> *pc-settings* lod-force-actor) 0 (-> dc lod-set max-lod))))
+                      (when (not (-> *pc-settings* ps2-lod-dist?))
+                        (set! lod-idx (minmax (-> *pc-settings* lod-force-actor) 0 (-> dc lod-set max-lod)))
+                        (when (> (-> dc force-lod) 0))
```

Comment:
I think this should be a `-1` instead of `0`, since `-1` means "don't use force lod"

Reply:
yep, good catch.